supertokens_python/recipe/session/framework/flask/__init__.py

```diff
@@ -35,15 +36,18 @@ def verify_session(
     ] = None,
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[[_T], _T]:
-    if user_context is None:
-        user_context = {}
+    _ = user_context
 
     def session_verify(f: _T) -> _T:
         @wraps(f)
         def wrapped_function(*args: Any, **kwargs: Any):
+            nonlocal user_context
             from flask import make_response, request
 
             baseRequest = FlaskRequest(request)
+            if user_context is None:
+                user_context = default_user_context(baseRequest)
```

Comment:
this is wrong. We want to modify the user_context even if it's passed by the user to add the default stuff into their passed in user_context.

Reply:
Updated